_analyzers/token-filters/asciifolding.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: ASCIIFolding
+parent: Token filters
+nav_order: 20
+---
+
+# ASCIIFolding token filter
```

Comment:
Let's go with 2 words everywhere: "ASCII foldidng"

Reply:
@kolchfa-aws thats updated now, thank you for review